test/tools/repo.js

```diff
@@ -6,15 +9,14 @@ export {
 /**
  * Create an empty repo
  */
-function createEmpty (sh, path) {
-  sh.mkdir(path);
-  sh.cd(path);
-  sh.exec('npm init --force --yes');
+function createEmpty (path) {
+  fs.mkdirSync(path, { recursive: true });
+  childProcess.spawnSync('npm', ['init', '--force', '--yes'], { cwd: path });
```

Comment:
Tests may failed caused here.
`childProcess.spawnSync` has [problem on windows](https://github.com/nodejs/node-v0.x-archive/issues/2318), and has [option](https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options) for it.

Apply this to fix.
```suggestion
  childProcess.spawnSync('npm', ['init', '--force', '--yes'], { cwd: path, shell: true });
```

Anyway, I see many outputs from npm command, but I don't know how to silent these...
Somehow `{ stdio: 'ignore' }` not worked for me(Windows 10) only while test ¯\_(ツ)_/¯

Reply:
Thank you!!